Kconfig

```diff
@@ -1,28 +0,0 @@
-# SPDX-License-Identifier: GPL-2.0-only
```

Comment:
Why did you remove this file?

Reply:
We move tls module out of linux kernel source so we dont have cmd like `make menuconfig`. For simplification I moved the config to Makefile directly.

Reply:
Hmmm, that's an assumption that doesn't always hold. For instance a user could clone this repository in-tree and integrate to the in-tree Kbuild system.
Let's re-introduce this file.

Reply:
reverted